callback_plugins/custom_junit.py

```diff
@@ -6,6 +6,32 @@
 import re
 from ansible.utils._junit_xml import TestCase, TestError, TestFailure, TestSuite, TestSuites
 
+DOCUMENTATION = '''
+    callback: custom_junit
+    type: notification
+    short_description: TODO
+    description:
+      TODO
```

Comment:
custom_junit generates an XML file that polarion can read.
Only the tasks marked with $test_case_prefix are reported.
The first line of the task name (excluding the prefix) is converted to snake_case, and this becomes the testcase name in the results file.